--- links-current/terminal.c.pix	2002-05-10 22:16:20.000000000 +0200
+++ links-current/terminal.c	2002-05-16 14:19:13.000000000 +0200
@@ -501,7 +501,7 @@
 
 struct list_head term_specs = {&term_specs, &term_specs};
 
-struct term_spec dumb_term = { NULL, NULL, "", 0, 1, 0, 0, 0, 0 };
+struct term_spec dumb_term = { NULL, NULL, "", 1, 1, 0, 0, 1, 0 };
 
 struct term_spec *get_term_spec(unsigned char *term)
 {
